apps/web/screens/UniversitySearch/Comparison.tsx

```diff
@@ -0,0 +1,337 @@
+              <T.Data>
+                <Text variant="eyebrow">
+                  {n('durationInYears', 'Lengd náms')}
+                </Text>
+              </T.Data>
+              {selectedComparison.map((i) => {
+                return (
+                  <T.Data>{`${i.durationInYears} ${
+                    locale === 'en'
+                      ? i.durationInYears === 1
+                        ? 'year'
+                        : 'years'
+                      : 'ár'
+                  }`}</T.Data>
+                )
+              })}
+            </T.Row>
+          </T.Body>
+          <T.Foot>
+            <T.Row>
+              <T.Data borderColor="transparent"></T.Data>
+              {selectedComparison.map((i) => {
+                const now = new Date()
+                return (
+                  <T.Data borderColor="transparent">
+                    <Box
+                      width="full"
+                      display="flex"
+                      flexDirection="column"
+                      rowGap={1}
+                    >
+                      {new Date(i.applicationStartDate) <= now &&
+                        new Date(i.applicationEndDate) >= now && (
+                          <Button size="small" fluid>
+                            {n('apply', 'Sækja um')}
+                          </Button>
+                        )}
+
+                      <LinkV2
+                        href={
+                          linkResolver('universitysearchdetails', [i.id]).href
+                        }
+                        passHref
+                      >
+                        <Button size="small" variant="ghost" fluid>
+                          {n('previewProgram', 'Skoða nám')}
+                        </Button>
+                      </LinkV2>
+                    </Box>
+                  </T.Data>
+                )
+              })}
+            </T.Row>
+          </T.Foot>
+        </T.Table>
+      </Box>
+    </GridContainer>
+  )
+}
+
+Comparison.getProps = async ({ query, apolloClient, locale }) => {
+  const { comparison } = query
+  const parsedComparison = JSON.parse(
+    !!comparison ? (comparison as string) : '',
+  )
+
+  const namespaceResponse = await apolloClient.query<
+    GetNamespaceQuery,
+    GetNamespaceQueryVariables
+  >({
+    query: GET_NAMESPACE_QUERY,
+    variables: {
+      input: {
+        lang: locale,
+        namespace: 'universityGateway',
+      },
+    },
+  })
+
+  const namespace = JSON.parse(
+    namespaceResponse?.data?.getNamespace?.fields || '{}',
+  ) as Record<string, string>
+
+  let showPagesFeatureFlag = false
+
+  if (publicRuntimeConfig?.environment === 'prod') {
+    showPagesFeatureFlag = Boolean(namespace?.showPagesProdFeatureFlag)
+  } else if (publicRuntimeConfig?.environment === 'staging') {
+    showPagesFeatureFlag = Boolean(namespace?.showPagesStagingFeatureFlag)
+  } else {
+    showPagesFeatureFlag = Boolean(namespace?.showPagesDevFeatureFlag)
```

Comment:
just curious, so the FF is set up within the namespace? not configcat?

Reply:
@RunarVestmann can probably answer this, there is some problem with using configCat feature flags on the web now after the last monorepo update, this is hopefully just temporary

Reply:
The feature flag functionality that was on the web previously wasn't available anymore after a configcat change in the monorepo and feature flag functionality still hasn't been added to the web. So I suggested as a workaround to perhaps use a namespace and nextconfig to determine whether to display the pages or not. However I'm a bit concerned regarding the 404 redirect that's on prod at the moment. Once this code goes live then island.is/haskolanam will no longer redirect users to /s/haskolanam

Reply:
Because the 404 redirects kick in if a page doesn't exist but since we've now added /haskolanam to the pages folder it'll appear instead (and just display a 404). However we could add a redirect in the infra repo instead which should take care of that issue